spec/julia-spec.coffee

```diff
@@ -195,7 +195,7 @@ describe "Julia grammar", ->
     foo bar
         \"\"\"""")
     expect(tokens[0]).toEqual value: "\"\"\"", scopes: ["source.julia", "string.docstring.julia", "punctuation.definition.string.begin.julia"]
-    expect(tokens[1]).toEqual value: "\ndocstring\n\nfoo bar", scopes: ["source.julia", "string.docstring.julia", "source.gfm"]
+    expect(tokens[1]).toEqual value: "\ndocstring\n\nfoo bar", scopes: ["source.julia", "string.docstring.julia", "text.nd"]
```

Comment:
Typo here. I think this should be `text.md` (See travis failure)

Reply:
And that's what happens if you make changes using GitHub and are to lazy to run the specs locally...
PR should probably be squashed on merge so all the unnecessary commits don't show up.